graph_slam:
  ros__parameters:
    perception_topic: "/perception/map"
    car_state_topic: "/car_state/state"
    lap_count_topic: "/slam/lap_count"
    map_topic: "/slam/map"
    final_map_topic: "/slam/final_map"
    global_frame: "arussim/world"
    local_frame: "slam/vehicle"

    finish_line_offset: 0.0
    track_width: 3.0
    min_lap_distance: 30.0
    max_pose_edges: 2500
    max_landmark_edges: 5000
    pos_lidar_x: 1.5

    debug: true
